.circleci/config.yml

```diff
@@ -183,7 +183,7 @@ workflows:
           context: orb-publishing
           requires: [queue/block_workflow]
 
-  integration_tests-prod_deploy:
+  integration-test_deploy:
```

Comment:
For consistency I think this should be `integration_test-deploy` - `-` is already used as a "step" separator in `test-pack`.